pom.xml

```diff
@@ -2525,7 +2525,7 @@
               <fork>true</fork>
               <failOnError>false</failOnError>
               <meminitial>1024m</meminitial>
-              <maxmem>8192m</maxmem>
```

Comment:
Do not reduce maxmem here. We want to allow checker to consume as much memory as possible, while restricting the build system's memory.

Reply:
this update was for testing only.

Reply:
> Do not reduce maxmem here. We want to allow checker to consume as much memory as possible, while restricting the build system's memory.

will revert in next push, was for testing. i will instead proceed with an infinite loop and capture the exit code, and see where it goes from there.

Reply:
> this update was for testing only.

Yep, I am aware of this.

>We want to allow checker to consume as much memory as possible, while restricting the build system's memory.

Reply:
> an infinite loop 

I am not sure what this would get us, other than wasting a massive amount of CI resources and tying up Github actions.